pyexchange/dydx.py

```diff
@@ -62,12 +62,14 @@ def buy_to_sell_price(self) -> Wad:
 
     @property
     def remaining_buy_amount(self) -> Wad:
-        return self.amount * self.price if self.is_sell else self.amount
+        amount = Wad.from_number(from_wei(abs(int(float(self.amount))), 'ether'))
```

Comment:
Where is `from_wei` defined?

Reply:
good catch, added the import from eth_utils